fe/fe-core/src/main/java/org/apache/doris/service/arrowflight/DorisFlightSqlProducer.java

```diff
@@ -90,7 +90,7 @@
 
 /**
  * Implementation of Arrow Flight SQL service
- *
+ * <p>
```

Comment:
what the `<p>` means ?

Reply:
IDEA format code automatically added
It seems that because the comment is segmented。。